sia-react-native/screens/Events/api.js

```diff
@@ -0,0 +1,64 @@
+import moment from 'moment';
+import Constants from 'expo-constants';
+import uuid from 'uuid'
+
+const { manifest } = Constants;
+const api = manifest.packagerOpts.dev
+    ? manifest.debuggerHost.split(`:`).shift().concat(`:19010`)
+    : `productionAPI.example.com`;
```

Comment:
What is this API you are using? 
We have our own Backend component that acts as an events api. Use that.